tests/integration/targets/test_proxysql_query_rules_fast_routing/tasks/test_delete_using_check_mode.yml

```
@@ -0,0 +1,30 @@
+---
+- name: "{{ role_name }} | test_delete_using_check_mode | set current test"
+  set_fact:
+    current_test: test_delete_using_check_mode
+
+- include_tasks: base_test.yml
+
+### then
+
+- name: "{{ role_name }} | {{ current_test }} | check if delete query rule fast routing in check mode reported a change"
+  assert:
+    that:
+      - status is changed
```

```suggestion
      - status is changed
```
As was spotted, `check_mode` should report the state changed only if the state is going to change in actual (`check_mode: no`) mode.